release_constants.json

```diff
@@ -0,0 +1,44 @@
+{
+    "RELEASE_SUMMARY_FILEPATH": "../release_summary.md",
```

Comment:
I think just `FILEPATH` should be good to go, right - we don't want to specify complete since it is in json, but using relative here will make it confusing for other scripts where it is used.